bot_file.py

```diff
@@ -49,15 +49,16 @@ def __init__(self, *args, **kwargs):
     def _actually_parse_message(self, message):
         GenericSlackBot._actually_parse_message(self, message)
 
-        who_wants_it = None
+        people_who_want_notification = None
         try:
-            who_wants_it = self.notify.who_wants_it(message['text'])
+            people_who_want_notification = self.notify.who_wants_it(message['text'])
         except Exception as e:
             pass
 
-        if who_wants_it is not None:
-            for person in who_wants_it:
-                self.time_to_show(message['channel'], person)
+        if people_who_want_notification is not None:
+            for person in people_who_want_notification:
+                self.ping_person(message['channel'], person)
+
 
     def parse_direct_message(self, message):
 
```

Comment:
noticed that `now_statement` has the same doc comment as `wait_statement` way down↓↓↓